### PR TITLE
Fixing basic traffico countries, to be improved.

### PR DESCRIPTION
**Pull request by peterneubauer**

### CODE REVIEW

File: js/id/renderer/mapillary_sign_layer.js [hunk 1]
```diff
@@ -9,9 +9,20 @@ iD.MapillarySignLayer = function(context) {
         currentImage,
         svg, image_preview_div, request, signs_defs;
 
-    request = d3.json('css/traffico-release-0.1.5/global-patched.json', function(error, data) {
+    var signs_defs={};
```
Comment (pnorman) [inline js/id/renderer/mapillary_sign_layer.js]: lint error `12:9   error  signs_defs is already defined             no-redeclare`

File: js/id/renderer/mapillary_sign_layer.js [hunk 2]
```diff
@@ -9,9 +9,20 @@ iD.MapillarySignLayer = function(context) {
...
+    d3.json('css/traffico-release-0.1.17/string-maps/de-map.json', function(error, data) {
         if (error) return;
-        signs_defs = data;
+        signs_defs['eu'] = data;
```
Comment (pnorman) [inline js/id/renderer/mapillary_sign_layer.js]: lint error `15:9   error  ["eu"] is better written in dot notation  dot-notation`

File: js/id/renderer/mapillary_sign_layer.js [hunk 3]
```diff
@@ -9,9 +9,20 @@ iD.MapillarySignLayer = function(context) {
...
+    });
+
+    _.each(['au', 'br', 'ca', 'us'], function(el) {
+        d3.json("css/traffico-release-0.1.17/string-maps/"+el+"-map.json", function(error, data) {
```
Comment (pnorman) [inline js/id/renderer/mapillary_sign_layer.js]: two `error  Strings must use singlequote` lint errors

**Comment by pnorman** [comment]
also some `Unexpected console statement` lint errors

**Comment by peterneubauer** [comment]
Is this more ok @pnorman ?

**Comment by pnorman** [comment]
it's passing tests now - I haven't reviewed the changes in detail, but looks okay

**Comment by bhousel** [comment]
pulled relevant commits into a9389b6 on #2720

**Pull request closed without merging**